conanfile.py

```diff
@@ -15,15 +15,23 @@ class RestinioConan(ConanFile):
     )
 
     settings = "os", "compiler", "build_type", "arch"
-    options = {'boost_libs': ['none', 'static', 'shared'], 'use_openssl': ['false', 'true']}
-    default_options = {'boost_libs': 'none', 'use_openssl': 'false'}
+    options = {'boost_libs': ['none', 'static', 'shared'],
+               'use_openssl': ['false', 'true'],
+               'fmt_header_only': ['false', 'true']}
+    default_options = {'boost_libs': 'none',
+                       'use_openssl': 'false',
+                       'fmt_header_only': 'false'}
```

Comment:
Please note the _default_ I selected is not inline the RESTinio project.

Looking for feedback =)